README.md

```diff
@@ -14,6 +14,7 @@ None
 | Name              | Description |
 |-------------------|-------------|
 | PACKER_VERSION    | The version of [Packer](https://packer.io) to use. |
+| SHFMT_VERSION | The version of [shfmt](https://github.com/mvdan/sh#shfmt) to use. |
```

Comment:
This isn't specific to this PR, but I noticed that there is a discrepancy between teh language we use here and in `action.yml`.  Here we say "The version of X to use.", but in `action.yml` we say "The version of X to download and use."  Do we care?

Reply:
I think it would be best to be consistent. I think we should pick whichever we prefer and use it uniformly.

Reply:
I'm all for consistency.  I think "The version of X to use." is sufficient- the "download" bit doesn't really add much value IMHO.

Reply:
I agree with @dav3r.  See commit e5f90be.